src/zenml/orchestrators/utils.py

```diff
@@ -54,7 +69,7 @@ def create_tfx_pipeline(
     # step is executed (see `BaseOrchestrator.run_step(...)`)
     return tfx_pipeline.Pipeline(
         pipeline_name=zenml_pipeline.name,
-        components=tfx_components,  # type: ignore[arg-type]
+        components=list(tfx_components.values()),  # type: ignore[arg-type]
```

Comment:
Why was this neccessary? :-)

Reply:
Always needed a list, I just changed it above to a `Dict` so I could more easily fetch the components to add the manual upstream steps.